File: java/src/main/java/com/robwilliamson/mailfamiliar/MailfamiliarApplication.java
package com.robwilliamson.mailfamiliar;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableAsync;

@EnableAsync
@SpringBootApplication
public class MailfamiliarApplication {

  public static void main(String[] args) {
    SpringApplication.run(MailfamiliarApplication.class, args);
  }
}

File: kotlin/src/main/kotlin/com/robwilliamson/mailfamiliar/MailfamiliarApplication.kt
package com.robwilliamson.mailfamiliar

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
class MailfamiliarApplication

fun main(args: Array<String>) {
    runApplication<MailfamiliarApplication>(*args)
}
